Library/Homebrew/tap_migrations.rb

```diff
@@ -138,7 +132,8 @@
   "lilypond" => "homebrew/tex",
   "lmutil" => "homebrew/binary",
   "magit" => "homebrew/emacs",
-  "meld" => "homebrew/x11",
+  "mat" => "homebrew/boneyard",
```

Comment:
This line needs fixing, we moved `mat` back to the core. I presume this is a rebase gone slightly wrong or something.